android/src/main/kotlin/clean/news/ui/item/share/ItemShareIntent.kt

```diff
@@ -0,0 +1,48 @@
+package clean.news.ui.item.share
+
+import android.content.Intent
+import clean.news.core.entity.Item
+import clean.news.flow.WithComponent
+import clean.news.flow.WithIntent
+import clean.news.presentation.navigation.NavigationFactory.ItemShareKey
+import clean.news.ui.item.detail.ItemDetailScreen
+import clean.news.ui.item.detail.ItemDetailScreen.ItemDetailComponent
+import dagger.Module
+import dagger.Provides
+import dagger.Subcomponent
+import flow.ClassKey
+import flow.TreeKey
+import nz.bradcampbell.paperparcel.PaperParcel
+import nz.bradcampbell.paperparcel.PaperParcelable
+
+@PaperParcel
+class ItemShareIntent(val item: Item) : ClassKey(),
+		TreeKey,
+		ItemShareKey,
+		WithIntent,
+		WithComponent<ItemDetailComponent>,
+		PaperParcelable {
+
+	override fun getParentKey() = ItemDetailScreen(item)
+
+	override fun createComponent(parent: ItemDetailComponent) = parent.plus(
```

Comment:
Looks like our codestyle has it set to 100, and this line goes to 102. Will remove the wrapping.